data/outfits.txt

```diff
@@ -389,6 +389,15 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand to hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
+outfit "Laser Pistol"
+	category "Hand to Hand"
+	cost 4000
+	thumbnail "outfit/sidearm"
+	"capture attack" .3
+	"capture defense" .4
+	"unplunderable" 1
+	description "A high quality sidearm that offers increased protection for less than a rifle, it is often times used to dual wield it and a standard sidearm. The result of this action varies, but generally don't end up well."
```

Comment:
The last sentence is ungrammatical: "The result of this action varies, but generally don't end up well."

You either need to change it to be all plural ("The **results** of this action **vary**, but generally don't end ~~up~~ well.") or all singular ("The result of this action varies, but generally **doesn't** end ~~up~~ well.").

Reply:
woops. missed that, thanks